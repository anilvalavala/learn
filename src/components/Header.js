import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <div>
        <h1>My Business</h1>
        <NavLink activeClassName='isActive' to='/' exact={true}>Home</NavLink>
        <NavLink activeClassName='isActive' to='/orders'>Orders</NavLink>
        <NavLink activeClassName='isActive' to='/products'>Products</NavLink>
        <NavLink activeClassName='isActive' to='/contacts'>Contacts</NavLink>
    </div>
);

export default Header;