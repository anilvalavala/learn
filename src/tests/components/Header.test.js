import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('Test if Header component is rendered correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();

    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
})