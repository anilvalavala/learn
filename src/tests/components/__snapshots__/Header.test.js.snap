// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if Header component is rendered correctly 1`] = `
<div>
  <h1>
    My Business
  </h1>
  <NavLink
    activeClassName="isActive"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="isActive"
    to="/orders"
  >
    Orders
  </NavLink>
  <NavLink
    activeClassName="isActive"
    to="/products"
  >
    Products
  </NavLink>
  <NavLink
    activeClassName="isActive"
    to="/contacts"
  >
    Contacts
  </NavLink>
</div>
`;
